src/librustc_trans/abi.rs

```diff
@@ -440,6 +442,12 @@ impl CastTarget {
                 cx.data_layout().aggregate_align
                     .max(a.align(cx))
                     .max(b.align(cx))
+            },
+            CastTarget::ChunkedPrefix { chunk, .. } => {
+                cx.data_layout().aggregate_align
+                    .max(Reg { kind: RegKind::Integer, size: chunk }.align(cx))
+                    .max(Reg { kind: RegKind::Float, size: chunk }.align(cx))
+                    .max(Reg { kind: RegKind::Vector, size: chunk }.align(cx))
```

Comment:
I'd rather iterate the array and set 3 bool flags - some of these might not be used.